.travis.yml

```diff
@@ -15,6 +15,6 @@ before_install:
 
 script:
 - make lint
-- make ci-build
 - make ci-test
-- ./formicactl/formicactl
+- make ci-build
```

Comment:
I've put testing first because it feels right in my head, no other reason :dancer: